README.md

```diff
@@ -1 +1,39 @@
-# Hello-world
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-world
+
+> The goal here is to master all of the tools and best practices we learned about github flow
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
+
+## Authors
+
+👤 **Author1**
+
```

Comment:
- [ ] [OPTIONAL] Please change this to be your name instead.

Reply:
@ShadyShawkat thank you for your response.